examples/async_summarize.py

```diff
@@ -0,0 +1,94 @@
+"""Copyright 2019 REV
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import time
+from rev_ai import apiclient, JobStatus
+from rev_ai.models.asynchronous.summarization_job_status import SummarizationJobStatus
+from rev_ai.models.asynchronous.summarization_options import SummarizationOptions
+from rev_ai.models.nlp_model import NlpModel
+
+# String containing your access token
+access_token = "<your_access_token>"
+
+# Create your api client
+client = apiclient.RevAiAPIClient(access_token)
+
+# Submitting a job through a local file.
+#
+# job = client.submit_job_local_file("your_local_file_path",
+#                                    delete_after_seconds=2592000,
+#                                    language='en',
+#                                    summarization_config=SummarizationOptions(
+#                                        model=NlpModel.PREMIUM
+#                                    ))
+
+
+# Submitting a job with a link to the file you want transcribed
+# Change url to your url, custom_vocabularies is optional like above
+url = "https://www.rev.ai/FTC_Sample_1.mp3"
+job = client.submit_job_url(media_url=url,
+                            delete_after_seconds=2592000,
+                            language='en',
+                            summarization_config=SummarizationOptions(
+                                model=NlpModel.PREMIUM
+                            ))
+
+print("Submitted Job")
+
+while True:
+    # Obtains details of a job in json format
+    job_details = client.get_job_details(job.id)
+    status = job_details.status
+    summarization_status = job_details.summarization.status
+    print("Job Status : {}. Summarization status: {}.".format(status, summarization_status))
+
+    # Checks if the job has been transcribed and summarized. Please note that this is not the recommended way
+    # of getting job status in a real application. For recommended methods of getting job status
+    # please see our documentation on setting a callback url here:
+    # https://docs.rev.ai/resources/tutorials/get-started-api-webhooks/
+    if status == JobStatus.IN_PROGRESS or summarization_status == SummarizationJobStatus.IN_PROGRESS:
+        time.sleep(5)
+        continue
+
+    elif status == JobStatus.FAILED or summarization_status == SummarizationJobStatus.FAILED:
```

Comment:
```suggestion
    if status == JobStatus.FAILED or summarization_status == SummarizationJobStatus.FAILED:
```

Reply:
Just curious, why do you think this way is better? All examples follow the same pattern...

Reply:
The same reason you don't use `epse` for the next block. It is redundant.
The current code is correct as well